### PR TITLE
Normalize strings written to `TraceWriter`

### PR DESCRIPTION
**Pull request by ybiquitous**
This change aims to reduce the code using `strip` before passing a string to the `TraceWriter` class. (using `String#strip` internally)

### CODE REVIEW

**Comment by ybiquitous** [comment]
@yykamei This is WIP yet, could you please provide me your opinion?

**Comment by yykamei** [comment]
Useful. I like it :+1:

File: lib/runners/processor/tyscan.rb [hunk 1]
```diff
@@ -79,16 +77,12 @@ def tyscan_scan
...
-      stdout, stderr, status = capture3(nodejs_analyzer_bin, "scan", "--json", *args)
+      stdout, _stderr, status = capture3(nodejs_analyzer_bin, "scan", "--json", *args)
 
       # TyScan exited with 0 when finishing an analysis correctly.
       unless status.exitstatus == 0
-        return Results::Failure.new(guid: guid, message: <<~MESSAGE, analyzer: analyzer)
-          TyScan was failed with status #{status.exitstatus} since an unexpected error occurred.
-
-          STDERR:
-          #{stderr}
```
Comment (ybiquitous) [inline lib/runners/processor/tyscan.rb]: [note] STDERR is output to the trace, so I think better not to include it to the failure message.

**Comment by ybiquitous** [comment]
@yykamei I've fixed all the failed tests. Could you review again?

**Pull request merged**